packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -40,6 +40,13 @@ import { dontThrowNotFoundError } from './Collection'
  * @typedef {object} Stream
  */
 
+/**
+ * Document representing a io.cozy.oauth.clients
+ *
+ * @typedef {object} OAuthClient
+ * @property {string} _id - Id of the client
```

Comment:
Right this is probably a good idea. I based this type on the `FileDocument` definition above and didn't think about it that much.
But since I'm actually using the `_type` attribute in the methods it should appear here.